documentation/source/install/google.rst

```diff
@@ -0,0 +1,168 @@
+*************
```

Comment:
The name of doc/title  mention google cloud but the steps are for Amazon, we can modify this to appropriate title/content

Reply:
Please ignore this google.rst file. This was an old file added by mistake. Review not required for this.